SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+HLASM Language Support follows the Eclipse Vulnerability Reporting Policy. Vulnerabilities are tracked by the Eclipse security team, in cooperation with the HLASM Language Support project lead. Fixing vulnerabilities is taken care of by the HLASM Language Support project committers, with assistance and guidance of the security team.
```

Comment:
I do not think there is such position as "HLASM Language Support project lead". Do you mean Che4z lead?

Reply:
"HLASM Language Support project lead" is fine with me